loaders/src/glTF/2.0/Extensions/KHR_materials_transmission.ts

```diff
@@ -6,6 +6,258 @@ import { IMaterial } from "../glTFLoaderInterfaces";
 import { IGLTFLoaderExtension } from "../glTFLoaderExtension";
 import { GLTFLoader } from "../glTFLoader";
 import { IKHRMaterialsTransmission } from 'babylonjs-gltf2interface';
+import { Scene } from "babylonjs/scene";
+import { AbstractMesh } from "babylonjs/Meshes/abstractMesh";
+import { Mesh } from "babylonjs/Meshes/mesh";
+import { Texture } from "babylonjs/Materials/Textures/texture";
+import { RenderTargetTexture } from "babylonjs/Materials/Textures/renderTargetTexture";
+import { Observable } from "babylonjs/Misc/observable";
+
+interface ITransmissionHelperOptions {
+    /**
+     * The size of the render buffers
+     */
+    renderSize: number;
+}
+
+/**
+ *
+ */
+export class TransmissionHelper {
+
+    /**
+     * Creates the default options for the helper.
+     */
+    private static _getDefaultOptions(): ITransmissionHelperOptions {
+        return {
+            renderSize: 512
+        };
+    }
+
+    /**
+     * Stores the creation options.
+     */
+    private readonly _scene: Scene;
+    private _options: ITransmissionHelperOptions;
+    private _opaqueRTIndex: number = -1;
```

Comment:
Does this need to be a member? It's only used in one function.

Reply:
It's gone now.